pkg/kubelet/volumes.go

```diff
@@ -78,7 +78,7 @@ func (vh *volumeHost) NewWrapperCleaner(spec *volume.Spec, podUID types.UID) (vo
 	return c, nil
 }
 
-func (kl *Kubelet) newVolumeBuilderFromPlugins(spec *volume.Spec, podRef *api.ObjectReference, opts volume.VolumeOptions) (volume.Builder, error) {
+func (kl *Kubelet) newVolumeBuilderFromPlugins(spec *volume.Spec, podRef *api.ObjectReference, opts volume.VolumeOptions, secretClient volume.SecretClient) (volume.Builder, error) {
```

Comment:
@markturansky I read the comment, but it seems redundant.  I am all about the minimal required interface pattern, but I don't think it adds alot when you already have the object you need as a field of field of the object you're writing a function for.